README.md

```diff
@@ -98,6 +98,9 @@ Add the libname to either requirements.in or dev-requirents.in, then either upgr
 `pip-compile requirements.in > requirements.txt` or `make upgrade`
 `pip install -r requirements.txt`
 
+### Cleaning example code
+Before start creating you own apps, run the command `make clean_examples` in order to clean the example apps from the front and backend. It will also remove it's usage.
```

Comment:
There are a few typos here :)

"Before start creating **your** own apps, run the command `make clean_examples` in order to clean **up** the example apps from the front and backend. It will also remove **its** usage."

What do you mean by "it will also remove its usage", btw?

Reply:
done.

"it will also remove its usage" does not make sense anymore. It has been removed.